src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1937,7 +1937,7 @@ internal bool MethodIsCompatibleWithDelegateOrFunctionPointer(BoundExpression? r
                 //  * Every ref or similar parameter has an identity conversion to the corresponding target parameter
                 // Note the addition of the reference requirement: it means that for delegate type void D(int i), void M(long l) is
                 // _applicable_, but not _compatible_.
-                if (!hasConversion(this, delegateType.TypeKind, Conversions, delegateParameter.Type, methodParameter.Type, delegateParameter.RefKind, methodParameter.RefKind, ref useSiteInfo))
+                if (!hasConversion(delegateType.TypeKind, Conversions, source: delegateParameter.Type, destination: methodParameter.Type, sourceRefKind: delegateParameter.RefKind, destinationRefKind: methodParameter.RefKind, checkingReturns: false, ref useSiteInfo))
```

Comment:
>if (!hasConversion(delegateType.TypeKind, Conversions, source: delegateParameter.Type, destination: methodParameter.Type, sourceRefKind: delegateParameter.RefKind, destinationRefKind: methodParameter.RefKind, checkingReturns: false, ref useSiteInfo)) [](http://example.com/codeflow?start=16&length=250)

It looks like this line became too long to fit an average screen. Consider splitting it